src/odemis/dataio/test/csv_test.py

```diff
@@ -240,6 +240,29 @@ def testExportSpectrumLineNoWL(self):
             raised = True
         self.assertFalse(raised, 'Failed to read csv file')
 
+    def testExportTemporalSpectrum(self):
+        """Try simple temporal spectrum export; note that size is made up and is not specific"""
+        size = (150, 340)
+        dtype = numpy.uint16
+        md = {model.MD_WL_LIST: numpy.linspace(536e-9, 650e-9, size[0]).tolist(),
+              model.MD_ACQ_TYPE: model.MD_AT_TEMPSPECTRUM,
```

Comment:
If you want to test the time column, you'll need another metadata with model.MD_TIME_LIST (typically with about 1 ps per pixel). It's not necessary, as the function must also work without that metadata, but the typical usage will provide that metadata...